packages/backend/index.ts

```diff
@@ -50,10 +49,20 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(cors());
 app.use(limiter);
-app.use('/', indexRouter);
-app.use('/v1/crm', cors(), revertAuthMiddleware(), crmRouter);
-app.use('/v1/connection', cors(), revertAuthMiddleware(), connectionRouter);
-app.use('/v1/metadata', cors(), metadataRouter);
+app.use(versionMiddleware());
+
+// TODO: Just to test versions. Remove later
+const testv2Router = (_req: Request, res: Response) => {
```

Comment:
@Hvinder Is this `testRouter` required to test in a deployed env as well?

Reply:
Nvm, understood. Lets remove after the change is pushed.

Reply:
It's required to test if the versioning works on higher env. Might make sense to keep it in. I can send a response saying "v2 is WIP, use v1 APIs" or something similar if someone hits v2 on higher env.